react/features/welcome/components/WelcomePageSideBar.native.js

```diff
@@ -157,10 +157,12 @@ class WelcomePageSideBar extends Component<Props> {
  */
 function _mapStateToProps(state: Object) {
     const _localParticipant = getLocalParticipant(state);
+    const _localParticipantId = _localParticipant?.id;
+    const _displayName = _localParticipant && getParticipantDisplayName(state, _localParticipantId);
```

Comment:
This one won't have a value if there is no local participant, am I right? Won't it cause temporary flashes (weird re-renders?)

Reply:
It will first render nothing, then the actual text AFAICT. Now, note that is one is hard to catch, chances of the sidebar being open while there is no local participant are slim to none. However, the component is mounted and the backtrace happens in mapStateToProps because localParticipant is undefined and we were trying to access the id.